packages/input-select/src/Select.js

```diff
@@ -35,13 +35,21 @@ class Select extends React.Component {
           renderMenu={this._renderMenu}
           renderItem={this._renderItem}
           autoHighlight={false}
-          onMenuVisibilityChange={isMenuOpen => this.setState({ isMenuOpen })}
+          onMenuVisibilityChange={this.onMenuVisibilityChange}
           wrapperStyle={{}}
+          ref={ref => (this.input = ref)}
         />
       </DropdownWrapper>
     );
   }
 
+  onMenuVisibilityChange = isMenuOpen => {
+    if (!this.props.disableSearch && isMenuOpen) {
+      this.input.select();
```

Comment:
Add `&& this.input` to the if condition. Just to prevent an error if the ref is not set yet.

Reply:
done!